src/sentry/services/hybrid_cloud/log/service.py

```diff
@@ -43,28 +50,10 @@ def find_last_log(
         pass
 
 
-def impl_by_db() -> LogService:
-    from .impl import DatabaseBackedLogService
-
-    return DatabaseBackedLogService()
-
-
-def impl_by_outbox() -> LogService:
-    from .impl import OutboxBackedLogService
-
-    return OutboxBackedLogService()
-
-
 # An asynchronous service which can delegate to an outbox implementation, essentially enqueueing
 # delivery of log entries for future processing.
-log_service: LogService = silo_mode_delegation(
-    {
-        SiloMode.REGION: impl_by_outbox,
-        SiloMode.CONTROL: impl_by_db,
-        SiloMode.MONOLITH: impl_by_db,
-    }
-)
+log_service: LogService = cast(LogService, LogService.create_delegation())
 
 # A synchronous service which can delegate to a remote rpc endpoint.  Used by the outbox receiver
 # that handles rows generated by the asynchronous log_service implementation.
-log_rpc_service: LogService = cast(LogService, LogService.create_delegation())
+log_rpc_service: LogService = cast(LogService, LogService.create_delegation(force_remote=True))
```

Comment:
Ah hah!  Ok, I do like this, I think the part that took me the most time to understand is what `force_remote=True` is.  Would be reasonable to have a vanity method called `create_rpc_service`?  Or something like that?  It could be under the hood the same thing, but I was confused by how this hooked up.